crates/trycp_server/src/main.rs

```diff
@@ -237,19 +319,23 @@ fn save_file(file_path: PathBuf, content: &[u8]) -> Result<(), jsonrpc_core::typ
     Ok(())
 }
 
-fn get_info_as_json() -> String {
-    let output = Command::new("holochain")
-        .args(&["-i"])
-        .output()
-        .expect("failed to execute process");
+fn get_info_as_json() -> Value {
+    let output = Command::new("holochain").args(&["-i"]).output();
+    if output.is_err() {
+        return Value::String("failed to execute holochain".into());
+    }
+    let output = output.unwrap();
     let info_str = String::from_utf8(output.stdout).unwrap();
 
-    // poor mans JSON convert
```

Comment:
I would maintain that this is still a poor man's JSON convert, the comment was helpful to me in figuring out what's going on here and I think should stay